docker/build.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash -xe
 #########Set Me###############
-export PYVER=${PYVER:-2.7}
+export PYVER=${PYVER:-3.5}
 export PYV=${PYVER/./}
 export CUDA_VERSION=${CUDA_VERSION:-10}
 export NVIDIA_BUILD_ID=${NVIDIA_BUILD_ID:-12345}
 export CREATE_WHL=${CREATE_WHL:-YES}
 export CREATE_RUNNER=${CREATE_RUNNER:-NO}
 export DALI_BUILD_FLAVOR=${DALI_BUILD_FLAVOR}
+export BUILD_TYPE=${BUILD_TYPE:-Debug}
```

Comment:
I this Debug should not be a default.
Also you can list possible values in comment.
+Update the readme that new option is available.

Reply:
Done